apollo-ios/Sources/Apollo/GraphQLQueryWatcher.swift

```diff
@@ -35,16 +42,20 @@ public final class GraphQLQueryWatcher<Query: GraphQLQuery>: Cancellable, Apollo
   /// - Parameters:
   ///   - client: The client protocol to pass in.
   ///   - query: The query to watch.
+  ///   - refetchOnFailedUpdates: Should the watcher perform a network fetch when it's watched
+  ///     objects have changed, but reloading them from the cache fails. Defaults to `true`.
   ///   - context: [optional] A context that is being passed through the request chain. Defaults to `nil`.
   ///   - callbackQueue: The queue for the result handler. Defaults to the main queue.
   ///   - resultHandler: The result handler to call with changes.
   public init(client: ApolloClientProtocol,
               query: Query,
+              refetchOnFailedUpdates: Bool = true,
```

Comment:
@AnthonyMDev What do you think of `ApolloClient.watch` passing this? If you have opinions against it, I can skip calling a wrapped `ApolloClient` straight to use `GraphQLQueryWatcher` directly.

Reply:
Oh good call. I should definitely add that to this PR!